packages/react/src/lazy.ts

```diff
@@ -108,3 +109,126 @@ const createLazy =
 export const lazy = Object.assign(createLazy({}), {
   create: createLazy,
 })
+
+export interface ReloadOnErrorStorage {
```

Comment:
I thought this shouldn't be exported

```suggestion
interface ReloadOnErrorStorage {
```